torch/_inductor/scheduler.py

```diff
@@ -4153,24 +4290,73 @@ def resolve_pending_fusions(
             node1 = self.get_fused_node(node1)
             node2 = self.get_fused_node(node2)
 
+            if (
+                is_template_fusion(node1, node2)
+                and (node1, node2) in self.seen_template_fusions
```

Comment:
We allow more reordering on the final round. See: https://github.com/pytorch/pytorch/blob/d74f9ecce649b52c52bea857c1c7d1e859ff6826/torch/_inductor/scheduler.py#L4167-L4171

Not relevant for template, maybe relevant for epilogue. cc @shunting314 loop reordering

Reply:
it means the cache might be outdated